### PR TITLE
[FUSEQE-6595] Data-testid implementation

### PR DESCRIPTION
**Pull request by Stefan365**

### CODE REVIEW

**Comment by syndesisqe** [comment]
Checkstyle verification failed with:
```
ui-tests/src/test/java/io/syndesis/qe/pages/integrations/editor/add/connection/actions/google/
  EventForm.java:3:15: Unused import - com.codeborne.selenide.Selenide.$. [UnusedImports]
ui-tests/src/test/java/io/syndesis/qe/steps/customizations/connectors/detail/
  DetailSteps.java:17: Wrong order for 'io.syndesis.qe.pages.customizations.connectors.detail.ApiClientConnectorDetail' import. [ImportOrder]
```

**Comment by Stefan365** [comment]
@mcada please check this document 
https://docs.google.com/spreadsheets/d/1WWb8afWKbqd33-k8zl-3bcu1UdBhs2XKx_seCM4cBOU/edit?usp=sharing

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by Stefan365** [comment]
> Checkstyle verification failed with:
> 
> ```
> ui-tests/src/test/java/io/syndesis/qe/pages/integrations/editor/add/connection/actions/google/
>   EventForm.java:3:15: Unused import - com.codeborne.selenide.Selenide.$. [UnusedImports]
> ui-tests/src/test/java/io/syndesis/qe/steps/customizations/connectors/detail/
>   DetailSteps.java:17: Wrong order for 'io.syndesis.qe.pages.customizations.connectors.detail.ApiClientConnectorDetail' import. [ImportOrder]
> ```

done

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

File: ui-tests/src/test/java/io/syndesis/qe/steps/customizations/connectors/detail/DetailSteps.java [hunk 1]
```diff
@@ -4,27 +4,50 @@
...
     private ApiClientConnectorDetail connectorDetailPage = new ApiClientConnectorDetail();
 
-    @Then("^validate connector detail values")
-    public void editProperty(DataTable dataTable) {
+    @Then("^validate connector detail values by id")
+    public void validateById(DataTable dataTable) {
+        this.validateConnectorDetail(dataTable, "id");
+    }
+
+    @Then("^validate connector detail values by data-testid")
```
Comment (mmuzikar) [inline ui-tests/src/test/java/io/syndesis/qe/steps/customizations/connectors/detail/DetailSteps.java]: Those two steps could probably be merged into one with a parameter for which selectors are going to be used

Comment (Stefan365) [reply ui-tests/src/test/java/io/syndesis/qe/steps/customizations/connectors/detail/DetailSteps.java]: yes, indeed.

**Comment by Stefan365** [comment]
@mmuzikar all comments have been incorporated

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
Integration - DB to DB | Read and update operations 
Integration info | Check error for integration with the same name 
Integration info | Check error for integration which has same name after update 
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Pull request merged**